plugins/remotelog/plugin.go

```diff
@@ -80,28 +78,22 @@ func configure(_ *node.Plugin) {
 
 	getGitInfo()
 
-	workerPool = workerpool.NewNonBlockingQueuedWorkerPool(func(task workerpool.Task) {
-		deps.RemoteLogger.SendLogMsg(task.Param(levelIndex).(logger.Level), task.Param(nameIndex).(string), task.Param(blockIndex).(string))
-
-		task.Return(nil)
-	}, workerpool.WorkerCount(runtime.GOMAXPROCS(0)), workerpool.QueueSize(1000))
+	workerPool = workerpool.New("RemoteLog", 1)
```

Comment:
why not use the worker pool integrated in the plugin?

Reply:
Good catch